Sources/nest/Commands/RunCommand.swift

```diff
@@ -0,0 +1,174 @@
+import ArgumentParser
+import Foundation
+import NestKit
+import NestCLI
+import Logging
+
+struct RunCommand: AsyncParsableCommand {
+    static let configuration = CommandConfiguration(
+        commandName: "run",
+        abstract: "Run executable file on a given nestfile. If not found, it will attempt to install."
+    )
+
+    @Flag(name: .shortAndLong)
+    var verbose: Bool = false
+    
+    @Flag(help: "Will not perform installation.")
+    var noInstall: Bool = false
+    
+    @Option(help: "A nestfile written in yaml.")
```

Comment:
That nestfile is written in yaml  is obvious.
```suggestion
    @Option(help: "A path to nestfile")
```

Reply:
Thanks!
I've made the corrections here. (https://github.com/mtj0928/nest/pull/36/commits/6f71256e1c3c3b6fad544204e737fa9a90bbf34e)